gax/src/error/error.rs

```diff
@@ -63,8 +63,19 @@ impl Error {
     /// Recurses through the source error chain and returns a reference to the
     /// inner value if it is of type `T`, or `None` if no such inner value is
     /// found.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use gax::error::Error;
+    /// # use gax::error::HttpError;
+    /// # use std::collections::HashMap;
+    /// let error: Error = HttpError::new(404, HashMap::new(), None).into();
```

Comment:
Is there a symbolic name for `404`?  though I wonder what is clearer... no action required.

Reply:
Kept as 404 so we don't need an import. I think this is clear enough for now.